solutions/src/main/java/ru/sberbank/school/task02/FxConversionServiceImpl.java

```diff
@@ -74,24 +79,62 @@ public BigDecimal convert(@NonNull ClientOperation operation, Symbol symbol, Big
      * @param beneficiary В чью пользу осуществляется округление
      * @return Цена для указанного объема
      */
+
     @Override
     public Optional<BigDecimal> convertReversed(@NonNull ClientOperation operation,
-                                         Symbol symbol,
-                                         BigDecimal amount,
-                                         Beneficiary beneficiary) {
+                                                @NonNull Symbol symbol,
+                                                @NonNull BigDecimal amount,
+                                                @NonNull Beneficiary beneficiary) {
         if (amount.compareTo(BigDecimal.ZERO) <= 0) {
             throw new IllegalArgumentException("Wrong amount");
         }
+        List<Quote> rightQuotes = createRightQuotes(exQuotes.getQuotes(symbol), amount, symbol, operation);
+        if (rightQuotes.isEmpty()) {
+            return Optional.empty();
+        }
+        BigDecimal revCur = operation == ClientOperation.BUY ?
```

Comment:
&#39;?&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-19752297>*